test/e2e/specs/test.js

```diff
@@ -8,12 +8,77 @@ module.exports = {
   'default e2e tests': function (browser) {
     browser
     .url(browser.launchUrl)
-      .waitForElementVisible('#app', 5000)
-      .assert.elementPresent('.randPersonBtn')
+      .waitForElementVisible(ui.dev.app, 5000)
+      .assert.elementPresent(ui.dev.randomBtn.person)
+      .pause(5000)
+      .end()
+  },
+
+  'verify person record has seven table records': function (browser) {
+    browser
+      .url(browser.launchUrl)
+      .waitForElementVisible(ui.dev.app, 5000)
+      .assert.elementPresent(ui.dev.randomBtn.person)
+      .assert.elementCount(ui.dev.tableRecord, 7)
```

Comment:
Done. Mostly. Groundwork for page objects is there. They aren't hooked up in the tests exactly as they should be. I embedded a css tag in every Vue element, I modeled after what we use at Fitbit: It is "[data-qa="nameTextbox"]' for a textbox on the Person page.